editor/editor_properties_array_dict.cpp

```diff
@@ -254,7 +254,7 @@ void EditorPropertyArray::update_property() {
 		array_type_name = vformat("%s[%s]", array_type_name, type_name);
 	}
 
-	if (array.get_type() == Variant::NIL) {
+	if (!array.is_array()) {
```

Comment:
Not sure what this change is about though.

Reply:
When you assign an int in place of the whole array since it is not a `Variant::NIL` it would just go on and assume it is an array try to get the length of it etc which are actions only accessible to variant types that are arrays hence the high amount of errors that are shown. Other places of this class already used `!array.is_array` for example in the `_edit_pressed` function so most likely there had already been problems of the sort and this one was missed / got merged after.